tools/ncf.py

```diff
@@ -513,9 +522,20 @@ def generate_technique_content(technique, methods):
     # Add commentary for each new line in the metadata
     metadata_value = technique[metadata_key].replace("\n", "\n# ")
     content.append('# @'+ metadata_key +" "+ metadata_value)
+
+  bundle_param = ""
+  if technique["parameter"] > 0:
```

Comment:
Why do you compare a list with an integer

Reply:
forgot a len ...